meta-mender-core/classes/mender-install.bbclass

```diff
@@ -35,6 +35,33 @@ MENDER_DEVICE_TYPE ?= "${MACHINE}"
 # Space separated list of device types compatible with the built update.
 MENDER_DEVICE_TYPES_COMPATIBLE ?= "${MACHINE}"
 
+# Total size of the medium that mender sdimg will be written to. The size of
```

Comment:
`The total size of the medium where the Mender sdimg will be written to. The size of the rootfs` I am surprised myself I am noticing this stuff now. It's because of @eysteins complaining about not using articles :)